test/strategies/Strategy.test.ts

```diff
@@ -16,8 +18,17 @@ import { testStrategyReapRewardExtra } from "./StrategyReapRewardExtra.test"
 import { testStrategyReapUninvestedReward } from "./StrategyReapUninvestedReward.test"
 import { testStrategyUpgradeable } from "./StrategyUpgradeable.test"
 import { testStrategyWithdraw } from "./StrategyWithdraw.test"
-import { Contract } from "ethers"
-import { execSync } from "child_process"
+
+testStrategyAccessControl
```

Comment:
I really don't think it's good to have this unless it's WIP code

Reply:
I even installed a VSCode on windows without any plugins and got the same behaviour: all imports for unused variables are automatically removed (I had the right settings in my project: don't remove unused settings)
I have to uncomment code many times and save it, and VSCode just removes those imports which is quite a productivity problem for me.

If it bothers you, I could move those to the end of the file.